### PR TITLE
afaryy/TDL08

### PR DESCRIPTION
**Pull request by afaryy**
Deliver to-do-list-08 codes.

### CODE REVIEW

File: TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+all : uploadconfigfile plan apply
+
+.PHONY: uploadconfigfile # Upload the config file to the bucket
+uploadconfigfile: 
+	if [ -d "upload" ]; then aws s3 sync upload s3://${BUCKET_NAME}/${PREFIX}; fi
+
+.PHONY: plan # run terraform plan
+plan:
+	terraform init;terraform refresh;terraform plan -out tfplan
```
Comment (hbwork) [inline TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/Makefile]: Run "terraform fmt" before terraform init?

File: TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/main.tf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+
+module "alarm" {
+  source                  = "./modules/cw-alarm-sns" 
+  sns_name                = var.sns_name
+  alarm_name              = var.alarm_name
+  metric_namespace        = var.metric_namespace
+  metric_name             = var.metric_name
+}
```
Comment (hbwork) [inline TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/main.tf:33]: The easy way to format and check your terraform code is 'terraform fmt'

File: TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/Makefile (hunk 1 above)
Comment (afaryy) [reply TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/Makefile]: Ok, thanks.

File: TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/main.tf (hunk 2 above)
Comment (afaryy) [reply TDL-08-EC2-Cloudwatch-alarm-SNS/terraform-code/main.tf:33]: OK

**Comment by afaryy** [comment]
Add terraform fmt in Makefile and modify policy in iam.tf to PutMetricData.

**Pull request merged**